lib/sentry.dart

```diff
@@ -20,6 +19,9 @@ import 'src/version.dart';
 
 export 'src/version.dart';
 
+/// Used to provide time-stamp for logging.
```

Comment:
nit: [according to Wikipedia](https://en.wikipedia.org/wiki/Timestamp) "timestamp" is the correct spelling :nerd_face:

Reply:
Done.